testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1136,7 +1136,9 @@ public void testCreateSchema()
     public void testDropNonEmptySchema()
     {
         if (!supportsCreateSchema()) {
-            assertQueryFails("DROP SCHEMA " + getSession().getSchema().orElseThrow(), "This connector does not support dropping schemas");
+            assertQueryFails("DROP SCHEMA " + getSession().getSchema().orElseThrow(), "(?si).*" +
```

Comment:
We should try to drop some temporary schema instead. As we know it is a dangerous operation when it is called on testing instance that is reused between tests.

Reply:
If connector does not support creating schema then there is no way to create temp schema (and it still shouldn't drop this schema as it's not empty)

Reply:
@kokosing it was written this way purposefully.